internal/scaling/scaling.go

```diff
@@ -323,82 +343,135 @@ func (e *Estimate) Markdown() []byte {
-		if ref.ContactSupport {
-			plus = "+"
-			note = "[contact support](mailto:support@sourcegraph.com)"
-		}
 
-		var replicas string
-		if ref.Replicas == def.Replicas {
-			replicas = fmt.Sprint(ref.Replicas, plus)
-		} else {
-			replicas = fmt.Sprint("**_", ref.Replicas, plus, "ꜝ_**")
-		}
+		var replicas = "n/a"
+		var cpuRequest = "n/a"
+		var cpuLimit = "n/a"
+		var memoryGBRequest = "n/a"
+		var memoryGBLimit = "n/a"
 
-		var cpuRequest string
-		if ref.CPU.Request == def.CPU.Request {
-			cpuRequest = fmt.Sprint(ref.CPU.Request, plus)
-		} else {
-			cpuRequest = fmt.Sprint("**_", ref.CPU.Request, plus, "ꜝ_**")
-		}
+		if !ref.ContactSupport {
+			if ref.Replicas == def.Replicas {
+				replicas = fmt.Sprint(ref.Replicas, plus)
+			} else {
+				replicas = fmt.Sprint(ref.Replicas, plus, "ꜝ")
+			}
 
-		var cpuLimit string
-		if ref.CPU.Limit == def.CPU.Limit {
-			cpuLimit = fmt.Sprint(ref.CPU.Limit, plus)
-		} else {
-			cpuLimit = fmt.Sprint("**_", ref.CPU.Limit, plus, "ꜝ_**")
-		}
+			if ref.CPU.Request == def.CPU.Request {
+				cpuRequest = fmt.Sprint(ref.CPU.Request, plus)
+			} else {
+				cpuRequest = fmt.Sprint(ref.CPU.Request, "ꜝ")
+			}
 
-		var memoryGBRequest string
-		if ref.MemoryGB.Request == def.MemoryGB.Request {
-			memoryGBRequest = fmt.Sprint(ref.MemoryGB.Request, "g", plus)
-		} else {
-			memoryGBRequest = fmt.Sprint("**_", ref.MemoryGB.Request, "g", plus, "ꜝ_**")
-		}
+			if ref.CPU.Limit == def.CPU.Limit {
+				cpuLimit = fmt.Sprint(ref.CPU.Limit, plus)
+			} else {
+				cpuLimit = fmt.Sprint(ref.CPU.Limit, plus, "ꜝ")
+			}
 
-		var memoryGBLimit string
-		if ref.MemoryGB.Limit == def.MemoryGB.Limit {
-			memoryGBLimit = fmt.Sprint(ref.MemoryGB.Limit, "g", plus)
-		} else {
-			memoryGBLimit = fmt.Sprint("**_", ref.MemoryGB.Limit, "g", plus, "ꜝ_**")
-		}
+			if ref.MemoryGB.Request == def.MemoryGB.Request {
+				memoryGBRequest = fmt.Sprint(ref.MemoryGB.Request, "g", plus)
+			} else {
+				memoryGBRequest = fmt.Sprint("", ref.MemoryGB.Request, "g", plus, "ꜝ")
+			}
 
-		if e.DeploymentType == "docker-compose" {
-			cpuRequest = "-"
-			memoryGBRequest = "-"
+			if ref.MemoryGB.Limit == def.MemoryGB.Limit {
+				memoryGBLimit = fmt.Sprint(ref.MemoryGB.Limit, "g", plus)
+			} else {
+				memoryGBLimit = fmt.Sprint(ref.MemoryGB.Limit, "g", plus, "ꜝ")
+			}
+
+			if e.DeploymentType == "docker-compose" {
+				cpuRequest = "-"
+				memoryGBRequest = "-"
+			}
 		}
 
 		fmt.Fprintf(
 			&buf,
-			"| %v | %v | %v | %v | %v | %v | %v |\n",
+			"| %v | %v | %v | %v | %v | %v |\n",
 			service,
 			replicas,
 			cpuRequest,
 			cpuLimit,
 			memoryGBRequest,
 			memoryGBLimit,
-			note,
 		)
 	}
 	fmt.Fprintf(&buf, "\n")
-	fmt.Fprintf(&buf, "_Bold/italic and ꜝ indicate the value is modified from the default. Services not listed here use the default values._")
+	fmt.Fprintf(&buf, "> ꜝ<small>: This is a non-default value.</small>\n")
+	fmt.Fprintf(&buf, "\n")
+
+	// Storage Size
+	fmt.Fprintf(&buf, "### Storage\n")
```

Comment:
For gitserver / indexed-search: is the size shown per replica, or in total? I'm guessing it's total and the customer will want to divide by the number of replicas when setting their PVC sizes in Kubernetes. Any concerns with that approach?

Reply:
I did not cross my mind so thanks for pointing this out! Would it help if we add the following footnote:
`This value represents the total disk space required for the associated service. Divide this by the number of replicas to get the value required for each replica.`
![image](https://user-images.githubusercontent.com/68532117/176000080-bd5a2eec-1bd4-4add-98d4-f567b0cc0ca5.png)